doc/calibration/adaptive-pressure-advance-calib.md

```diff
@@ -11,17 +11,23 @@ This feature introduces the below options under the filament settings:
-5. **Pressure advance:** The old field is still needed and is required to be populated with a PA value. A “good enough” median PA value should be entered here, as this will act as a fallback value when performing tool changes, printing a purge/wipe tower for multi-color prints as well as a fallback in case the model fails to identify an appropriate value (unlikely but it’s the ultimate backstop).
-
+5. **Pressure advance:** The old field is still needed and is required to be populated with a PA value. A “good enough” median PA value should be entered here, as this will act as a fallback value when performing tool changes, printing a purge/wipe tower for multi-color prints as well as a fallback in case the model fails to identify an appropriate value (unlikely but it’s the ultimate backstop).  
 ![apa-material-config](https://github.com/SoftFever/OrcaSlicer/blob/main/doc/images/pa/apa-material-config.png?raw=true)
 
 ## Pre-Requisites
 
-This feature has been tested with Klipper-based printers. While it may work with Marlin or Bambu lab printers, it is currently untested with them. It shouldn’t adversely affect the machine; however, the quality results from enabling it are not validated.
+This feature has been tested with Klipper-based printers. While it may work with Marlin or Bambu lab printers, it is currently untested with them. It shouldn't adversely affect the machine; however, the quality results from enabling it are not validated.
+
+> [!IMPORTANT]
+> Older versions of Klipper older than July 11th, 2024 might in some cases produce artifacts when dynamic PA is used.  
+> This has been fixed in Klipper stable version 0.13.0 (20250411), or in developer versions starting [from July 11th, 2024](https://github.com/Klipper3d/klipper/commit/c84d78f3f169bc5163d11b74837f9880b0b7dba4).  
+> If you experience issues or print quality lower than expected you might want to ensure you have a Klipper firmware which contains the fix.
 
-**Older versions of Klipper used to stutter when pressure advance was changed while the toolhead was in motion. This has been fixed with the latest Klipper firmware releases. Therefore, make sure your Klipper installation is updated to the latest version before enabling this feature, in order to avoid any adverse quality impacts.**
+> [!NOTE]
+> While adaptive pressure advance itself is a host-side feature that doesn't require MCU firmware updates to function, it's recommended to also update your printer's MCU firmware (main board and toolhead board if present) when upgrading Klipper for full compatibility with other potential improvements in the ecosystem.
```

Comment:
I fail to see the point of this remark. How to update, and what to update, should belong to Klipper's documentation, not Orca, which should be limited to directing to an update. 
The section "Advanced users" belo in my opinion provides all information which the user might need with the information (not obvious from Klipper's docs) that patching the host code is a viable solution.
Suggesting a MCU update WILL cause doubts in some people with printers with difficult to update toolheads (and I have a Qidi so I know) because they might think they are missing out something, or they might be causing future issues by not touching toolhead FW.

So with the next section for advanced users, I would delete this one.

Reply:
Also, suggesting toolhead update kind of contradicts the "only host code is needed"